src/components/templates/components/Avatar.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
```

Comment:
If you a used wrapper div, you should styled margin it, instead if img. Testing your code please.

Reply:
I see what you mean. Fixed now.